templates/bbb/bigbluebutton.j2

```diff
@@ -1,84 +1,78 @@
 server {
-        listen 80;
-        listen [::]:80;
-        server_name {{ bbb_hostname }};
-        return 301 https://$host$request_uri;
+    listen 80;
+    listen [::]:80;
+    server_name {{ bbb_hostname }};
+    return 301 https://$host$request_uri;
 }
 
-
 server {
-    listen 443 ssl;
-    listen [::]:443 ssl;
-    server_name {{ bbb_hostname }};
 
+  listen 443 ssl http2;
+  listen [::]:443 ssl http2;
+  server_name {{ bbb_hostname }};
     ssl_certificate {{ bbb_ssl_cert }};
     ssl_certificate_key {{ bbb_ssl_key }};
     ssl_session_cache shared:SSL:10m;
     ssl_session_timeout 10m;
-    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_protocols TLSv1.2;
```

Comment:
What about supporting TLSv1.3, is there anything that speaks against that?

Reply:
It would not work with v1.3 - ubuntu 16.04 LTS has OpenSSL version 1.0.2.
It required v1.1.1.
Source: https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_protocols

How do we manage it?

Reply:
Then I would say: leave it as is...